### PR TITLE
Change errors

### LINKED ISSUE #41: redo all errors to what is in controllers/auth.js

### PR DESCRIPTION
**Pull request by GoraJakub**

### CODE REVIEW

File: routes/owner.js [hunk 1]
```diff
@@ -25,30 +26,42 @@ router.post('/hotel', async (req, res) => {
...
     console.log(err)
-    res.status(500).send('Something went wrong')
+    throw new ApiError(500, 'Something went wrong')
   }
 })
 
-router.delete('/hotel/:id', async (req, res) => {
+router.delete('/hotel/:id&:force', async (req, res) => {
   const id = req.params.id
+  const force = req.params.force
```
Comment (hoolek77) [inline routes/owner.js]: u can use object destructuring
const { id, force} = req.params

**Review by hoolek77 — CHANGES_REQUESTED** [review]

File: routes/owner.js [hunk 2]
```diff
@@ -25,30 +26,42 @@ router.post('/hotel', async (req, res) => {
...
     res.status(200).send(hotel)
   } catch (err) {
     console.log(err)
-    res.status(500).send('Something went wrong')
+    throw new ApiError(500, 'Something went wrong')
   }
 })
 
-router.delete('/hotel/:id', async (req, res) => {
+router.delete('/hotel/:id&:force', async (req, res) => {
```
Comment (bartoszbialecki) [inline routes/owner.js]: I think it should be passed in query string

File: routes/owner.js [hunk 3]
```diff
@@ -25,30 +26,42 @@ router.post('/hotel', async (req, res) => {
...
   const id = req.params.id
+  const force = req.params.force
+  let message = ''
   try {
     const reservation = await Reservation.find({ hotelId: id })
 
-    if (reservation.length > 0)
-      return res.status(400).send('Remove reservations')
+    if (reservation.length > 0 && force === 'false')
```
Comment (bartoszbialecki) [inline routes/owner.js]: then force must be read from query string and be cast to boolean

File: routes/owner.js [hunk 4]
```diff
@@ -25,30 +26,42 @@ router.post('/hotel', async (req, res) => {
...
+    if (reservation.length > 0 && force === 'true') {
+      await Reservation.deleteMany({ hotelId: id })
+      message = 'and reservations '
+    }
 
     await Hotel.findByIdAndDelete(id)
-    res.status(200).send('Hotel deleted')
+
+    const hotels = await Hotel.find()
+    res.status(200).send({ message: `Hotel ${message}deleted`, hotels: hotels })
```
Comment (bartoszbialecki) [inline routes/owner.js]: Why do you return here list of hotels? I think we don't need it, in such situations when you change something I think is enough to return only information about success

**Review by bartoszbialecki — COMMENTED** [review]
I think that we also should work everywhere with json, so check what you get if you write res.send, I think we should use res.json

File: routes/owner.js (hunk 4 above)
Comment (GoraJakub) [reply routes/owner.js]: I had this in my issue

image

File: config/default.json [hunk 5]
```diff
@@ -1,5 +1,5 @@
 {
-  "jwtPrivateKey": "",
+  "jwtPrivateKey": "12345",
```
Comment (pdybowski) [inline config/default.json]: Remove value from this folder

**Review by pdybowski — COMMENTED** [review]
Do not merge your branches from two different prs. I don't want to send my comments millions of times. Moreover I don't know what changes we merge on what pr

**Pull request merged**